src/Helper/Placeholder/Container/AbstractContainer.php

```diff
@@ -117,14 +117,17 @@ public function toString($indent = null)
             ? $this->getWhitespace($indent)
             : $this->getIndent();
 
-        $items  = $this->getArrayCopy();
         // If we don't have items - do not show prefix and postfix
-        if (!count($items)) {
+        if (!$this->count()) {
```

Comment:
Please move this check before `$indent = ($indent !== null)` (line 116) and add a whitespace after `!`.